QueAns/views.py

```diff
@@ -0,0 +1,91 @@
+from django.http import HttpResponse, HttpResponseBadRequest, HttpResponseForbidden
+from .models import Voting,Satisfaction,Question,Meetings
+from django.utils import timezone
+
+
+def upvote(request,user_id,question_id):
+    if request.user.is_authenticated():
+        if request.method == "POST":
+            meeting = Question.objects.filter(id=question_id).values_list('meeting_id')
+            meets_end_time = Meetings.objects.filter(id=meeting).values_list('end_time')
+
+            if meets_end_time < timezone.now():
+                votes = Voting.objects.filter(user_id=user_id,question_id=question_id)
+                count = votes.len()
+                if count <1:
+                    Voting.objects.create(user_id=user_id,question_id=question_id,vote=1)
+                    return HttpResponse('Upvoted')
+                elif count==1:
+                    if votes.vote == 1:
+                        return HttpResponse('Already Upvoted')
+                    else:
+                        votes.vote = 2
+                        votes.save()
+                        return HttpResponse('Vote Changed')
```

Comment:
What is status of response here and what should be status of response according to HTTP specs.